_posts/2023-06-01-pride-month.md

```diff
@@ -17,6 +17,12 @@ We'll be publishing these posts each Monday during June to highlight such
 people, where we talk about the person and tie their work and life to the RSE
 movement.
 
+* [Sophie Wilson]({% post_url 2023-06-05-pride-spotlight-sophie-wilson %}) -
```

Comment:
Something about this is making CircleCI mad

Reply:
Pretty sure it's the "future dated" post. Can't create a link to something that doesn't exist. We'll have to make sure to only merge on/after 6/5.

Reply:
Yuuup. Makes sense.